app/controllers/TpsController.scala

```diff
@@ -124,20 +132,44 @@ class TpsController @Inject() (actions:        Actions,
     tpsPayments.copy(payments = tpsPaymentsListNew)
   }
 
-  private def maybeSendEmails(tpsPayments: TpsPayments)(implicit hc: HeaderCarrier): Unit = {
-    logger.debug("maybeSendEmails")
-    tpsPayments.payments.find(p => p.email.nonEmpty)
-      .fold(())(tpsPaymentItem => sendEmail(tpsPaymentItem))
-    ()
+  private def maybeSendEmail(tpsPayments: TpsPayments, status: model.StatusType)(implicit hc: HeaderCarrier): Unit = {
+    logger.debug("maybeSendEmail")
+    val tuple = (tpsPayments.payments.find(paymentItem => paymentItem.email.nonEmpty), status)
```

Comment:
This can be simplified 

val email = tpsPayments.payments.find(paymentItem => paymentItem.email.nonEmpty).flatMap(_.email)

Reply:
and then tuple up like

val emailStatus = (email, status)

Reply:
no because I need to use the language code from the same payment item, so don't need to just take the email at this point